docs/content/staking/custody-providers.mdx

```diff
@@ -22,7 +22,7 @@ Users using the Blocto wallet can also withdraw their rewards using the Flow Por
 ## Ledger
 
 If you're storing your locked FLOW tokens using a Ledger device, you can use the Flow Port to **stake** or **delegate** them to earn rewards.
-For more details you can read [our guides](https://docs.onflow.org/flow-port/) to familiarize yourself with the Flow Port app,
+For more details you can read [our guides](https://docs.onflow.org/flow-port/faq/) to familiarize yourself with the Flow Port app,
```

Comment:
```suggestion
For more details you can read [our guides](/flow-port/faq/) to familiarize yourself with the Flow Port app,
```

Reply:
I believe you can use relative path since the `faq` page is under `content/flow-port`